src/views/preview/presentation.jsx

```diff
@@ -87,7 +87,7 @@ const PreviewPresentation = ({
             { projectInfo && projectInfo.author && projectInfo.author.id && (
                 <Formsy onKeyPress={onKeyPress}>
                     <div className="inner">
-                        <FlexRow className="preview-row">
+                        <FlexRow className="preview-row force-row">
```

Comment:
@chrisgarrity I totally agree! This problem is a bigger one tho because there are so many flex rows everywhere and pretty often in the mobile dev prs I use a media query to force it to go to a row instead of column. I think fixing the mess should probably be addressed in a separate issue. What do you think?

Reply:
write up issue and add links to all lines that need to be fixed